docs/design/ovs-pipeline.md

```diff
@@ -155,22 +244,19 @@ spec:
       port: 80
 ```
 
-This Network Policy is applied to all Pods with the `nginx` app label in the
-`default` Namespace. For these Pods, it only allows TCP traffic on port 80 from
-and to Pods which also have the `nginx` app label. Because Antrea will only
-install OVS flows for this Network Policy on Nodes for which some of the Pods
-are the target of the policy, we have scheduled 2 `nginx` Pods on the same
-Node. They received IP addresses 10.10.1.2 and 10.10.1.3 from the Antrea CNI, so
-you will see these addresses show up in the OVS flows.
+This Kubernetes NetworkPolicy is applied to all Pods with the `nginx` app label in the `default` Namespace. For these
+Pods, it only allows TCP traffic on port 80 from and to Pods which also have the `nginx` app label. Because Antrea will
+only install OVS flows for this Kubernetes NetworkPolicy on Nodes for which some of the Pods are the target of the
+policy, we have scheduled 2 `app:nginx` Pods on the same Node. They received IP addresses `10.10.0.7` and `10.10.0.8`
+from the Antrea CNI, so you will see these IP addresses show up in the OVS flows.
 
-## Antrea-native Policies Implementation
+## Antrea-native NetworkPolicy Implementation
 
-In addition to the above tables created for K8s NetworkPolicy, Antrea creates
-additional dedicated tables to support the [Antrea-native policies](../antrea-network-policy.md)
-([AntreaPolicyEgressRuleTable] and [AntreaPolicyIngressRuleTable]).
+In addition to the above tables created for Kubernetes NetworkPolicy, Antrea creates additional dedicated tables to
+support the [Antrea-native NetworkPolicy](../antrea-network-policy.md) (table [AntreaPolicyEgressRule] and [AntreaPolicyIngressRule]).
```

Comment:
ditto about the links, there may be other places in the doc as well
doesn't the markdown lint checker give errors about these?